tests-behat/bootstrap/Context.php

```diff
@@ -242,7 +241,6 @@ public function iDontSee($arg1)
      */
     public function labelChangesToNumber()
     {
-        $this->getSession()->wait(5000, '!$("#' . $this->buttonId . '").hasClass("loading")');
```

Comment:
not needed, because we call `jQueryWait` after each step, ie. the first statement in a step should never check load - it should be always loaded